Microsoft.AspNetCore.Components/Microsoft.AspNetCore.Components/BindConverter.cs

```diff
@@ -0,0 +1,689 @@
+using System;
+using System.Globalization;
+
+namespace Microsoft.AspNetCore.Components
+{
+	/// <summary>
+	/// Performs conversions during binding.
+	/// </summary>
+	public static class BindConverter
+	{
+		private delegate object? BindFormatter<T>(T value, CultureInfo? culture);
+
+		private delegate object BindFormatterWithFormat<T>(T value, CultureInfo? culture, string format);
```

Comment:
Should this have a nullable return type as well? Feels weird that these two delegates would have different output types....